docs/examples/example_estimating_ates.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
Thanks a lot for adding an example! :)
For it to be shown properly in the docs, you'll have to add an entry for it in the corresponding index:
https://github.com/Quantco/metalearners/blob/main/docs/examples/index.rst